index.html

```diff
@@ -691,33 +691,17 @@ <h4>Proof Configuration (ecdsa-rdfc-2019)</h4>
 
           <ol class="algorithm">
             <li>
-Let <var>proofConfig</var> be an empty object.
+Let <var>proofConfig</var> be a clone of the <var>options</var> object.
             </li>
             <li>
-Set <var>proofConfig</var>.<var>type</var> to
-<var>options</var>.<var>type</var>.
-            </li>
-            <li>
-If <var>options</var>.<var>cryptosuite</var> is set, set
-<var>proofConfig</var>.<var>cryptosuite</var> to its value.
-            </li>
-            <li>
-If <var>options</var>.<var>type</var> is not set to `DataIntegrityProof` and
+If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` and
 <var>proofConfig</var>.<var>cryptosuite</var> is not set to `ecdsa-rdfc-2019`, an
```

Comment:
I'm not clear on what is a valid proof configuration, here, so not sure how to adjust phrasing. Is it that both `type` and `cryptosuite` have to be set as shown to be valid? Or is either one being set as shown sufficient to be valid? I'm thinking it's the former... which means this change should be applied --
```suggestion
If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` and/or
<var>proofConfig</var>.<var>cryptosuite</var> is not set to `ecdsa-rdfc-2019`, an
```

Reply:
Both must be set so taking your changes.

Reply:
This one (of three) doesn't seem to have been applied yet...

After this one's been applied, I'll approve this PR.